driver-sync/src/test/functional/com/mongodb/client/ClientSideEncryptionViewAreProhibitedTest.java

```diff
@@ -52,21 +51,12 @@ public void setUp() {
                         && !System.getProperty("org.mongodb.test.awsAccessKeyId").isEmpty());
 
         MongoClientSettings.Builder clientSettingsBuilder = getMongoClientSettingsBuilder();
-        CodecRegistry codecRegistry = fromRegistries(
-                fromCodecs(new UuidCodec(UuidRepresentation.STANDARD)),
-                MongoClientSettings.getDefaultCodecRegistry());
-        clientSettingsBuilder.codecRegistry(codecRegistry);
-        MongoClientSettings clientSettings = clientSettingsBuilder.build();
-        client = MongoClients.create(clientSettings);
+        MongoClient client = Fixture.getMongoClient();
 
         MongoDatabase db = client.getDatabase("db");
         db.getCollection("view").drop();
 
-        BsonDocument createCmd = new BsonDocument()
-                .append("create", new BsonString("view"))
-                .append("viewOn", new BsonString("coll"));
-
-        db.runCommand(createCmd, BsonDocument.class);
+        db.createView("view", "coll", asList(Document.parse("{$match: {}}")));
```

Comment:
Can this just be Collections.emptyList()?

Reply:
I will check it tomorrow.

Reply:
I used this one: `Collections.<BsonDocument>emptyList()` (`Collections.emptyList()` probably is not compliled)